src/app/forms/components/select/select.component.html

```diff
@@ -1,5 +1,5 @@
-<div *ngIf="isEditing; else view" [ngClass]="{ 'govuk-form-group': !noBottomMargin }">
-  <label *ngIf="label" class="govuk-label govuk-label--m" for="{{ name }}"> {{ label }} </label>
+<div [ngClass]="{ 'govuk-form-group': !noBottomMargin }">
+  <label *ngIf="label" class="govuk-label" for="{{ name }}"> {{ label }} </label>
```

Comment:
Why remove the `label--m` class? I think this is the label font modifier we use everywhere else.

Reply:
This resulted from my testing of using this inside of a table, not it is not necessary anymore. The change has been reverted - apologies for the confusion.